lib/vdc/CMakeLists.txt

```diff
@@ -66,11 +67,21 @@ set (HEADERS
 	${PROJECT_SOURCE_DIR}/include/vapor/DerivedVarMgr.h
 	${PROJECT_SOURCE_DIR}/include/vapor/DCUtils.h
 	${PROJECT_SOURCE_DIR}/include/vapor/QuadTreeRectangle.hpp
+	${PROJECT_SOURCE_DIR}/include/vapor/QuadTreeRectangleP.h
 )
 
 add_library (vdc SHARED ${SRC} ${HEADERS})
 
-target_link_libraries (vdc PUBLIC common wasp ${UDUNITS2} ${PROJ})
+if( USE_OMP )
+  target_compile_options( vdc PUBLIC ${OpenMP_CXX_FLAGS} )
+endif()
+
+
+if( USE_OMP )
+  target_link_libraries (vdc PUBLIC common wasp ${UDUNITS2} ${PROJ} OpenMP::OpenMP_CXX)
+else()
+  target_link_libraries (vdc PUBLIC common wasp ${UDUNITS2} ${PROJ})
```

Comment:
There is no need to duplicate the libraries linking calls. You can do:

```
link(vdc common wasp...)

if USE_OMP
    link(vdc OpenMP)
fi

Reply:
This is addressed in d2fe2762b.